src/main/resources/templates/message/list.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
```

Comment:
We were talking about freemarker actually, when we discussed project. So why I see thymeleaf?
And where is the form for adding new messages?